shared/constants/metametrics.js

```diff
@@ -278,23 +278,57 @@ export const REJECT_NOTFICIATION_CLOSE_SIG =
  */
 
 export const EVENT_NAMES = {
+  ACCOUNT_PASSWORD_CREATED: 'Account Password Created',
+  ACCOUNT_IMPORT_COMPLETED: 'Account Added',
+  ACCOUNT_IMPORT_FAILED: 'Account Add Failed',
+  ACCOUNT_RESET: 'Account Reset',
+  ADD_ACCOUNT_COMPLETED: 'Account Added',
+  ADD_ACCOUNT_FAILED: 'Account Add Failed',
   APP_INSTALLED: 'App Installed',
+  APP_UNLOCKED: 'App Unlocked',
+  APP_UNLOCKED_FAILED: 'App Unlocked Failed',
+  APP_WINDOW_EXPANDED: 'App Window Expanded',
   DECRYPTION_APPROVED: 'Decryption Approved',
   DECRYPTION_REJECTED: 'Decryption Rejected',
   DECRYPTION_REQUESTED: 'Decryption Requested',
-  ENCRYPTION_PUBLIC_KEY_APPROVED: 'Encryption Public Key Approved',
-  ENCRYPTION_PUBLIC_KEY_REJECTED: 'Encryption Public Key Rejected',
-  ENCRYPTION_PUBLIC_KEY_REQUESTED: 'Encryption Public Key Requested',
+  ENCRYPTION_PUBLIC_KEY_APPROVED: 'Encryption Approved',
+  ENCRYPTION_PUBLIC_KEY_REJECTED: 'Encryption Rejected',
+  ENCRYPTION_PUBLIC_KEY_REQUESTED: 'Encryption Requested',
+  EXTERNAL_LINK_CLICKED: 'External Link Clicked',
+  KEY_EXPORT_SELECTED: 'Key Export Selected',
+  KEY_EXPORT_REQUESTED: 'Key Export Requested',
+  KEY_EXPORT_FAILED: 'Key Export Failed',
+  KEY_EXPORT_CANCELED: 'Key Export Canceled',
+  KEY_EXPORT_REVEALED: 'Key Material Revealed',
+  KEY_EXPORT_COPIED: 'Key Material Copied',
+  NAV_ACCOUNT_MENU_OPENED: 'Account Menu Opened',
+  NAV_ACCOUNT_DETAILS_OPENED: 'Account Details Opened',
+  NAV_CONNECTED_SITES_OPENED: 'Connected Sites Opened',
+  NAV_MAIN_MENU_OPENED: 'Main Menu Opened',
+  NAV_NETWORK_MENU_OPENED: 'Network Menu Opened',
+  NAV_SETTINGS_OPENED: 'Settings Opened',
+  NAV_ACCOUNT_SWITCHED: 'Account Switched',
+  NAV_NETWORK_SWITCHED: 'Network Switched',
+  NAV_CREATE_ACCOUNT_BUTTON_CLICKED: 'Account Add Selected',
+  NAV_IMPORT_ACCOUNT_BUTTON_CLICKED: 'Account Add Selected',
+  NAV_CONNECT_HARDWARE_BUTTON_CLICKED: 'Account Add Selected',
```

Comment:
if you and @digiwand think it's important, i could add something like the following code snippet.

```
/**
 * EVENTS
 */

// Values below can be used in EVENT_NAMES to avoid misspellings when names are repeated
// Note: Deliberately not using 'export' as this should not be referred to outside this file
const SHARED_EVENT_NAMES = {
  ACCOUNT_ADDED: 'Account Added',
  ACCOUNT_ADD_SELECTED: 'Account Add Selected',
};

export const EVENT_NAMES = {
  ...
  NAV_CREATE_ACCOUNT_BUTTON_CLICKED: SHARED_EVENT_NAMES.ACCOUNT_ADD_SELECTED,
  NAV_IMPORT_ACCOUNT_BUTTON_CLICKED: SHARED_EVENT_NAMES.ACCOUNT_ADD_SELECTED,
  NAV_CONNECT_HARDWARE_BUTTON_CLICKED: SHARED_EVENT_NAMES.ACCOUNT_ADD_SELECTED,
```